vipps-terminology.md

```diff
@@ -0,0 +1,72 @@
+<!-- START_METADATA
+---
+title: Terminology
+sidebar_position: 95
+---
+END_METADATA -->
+
+# Vipps terminology
+
+Document version: 1.0.0.
+
+## Common terms
+
+* API keys - Codes that give a merchant access to the API (e.g., `client_id`, `client_secret`, `Vipps-Subscription-Key`).
+These are found in the [VippsPortal](https://portal.vipps.no/) in the *Utvikler* section, under *Show keys*.
+
+* BankID - Software that certifies the identity of a person.
+
+* D-number - A Norwegian identification number given to foreign persons who do not have a Norwegian national identity number, but have a need for identification for the Norwegian authorities.
+
+* Deeplink - The Vipps "deeplink" opens the payment page in Vipps where the user
+accepts a payment. This is an integrated part of the Vipps payment process,
+and the link should never be sent in an SMS or email.
+
+* Direct capture - **Direct capture** causes all payment reservations to be instantly be captured.
+This is intended for situations where the product or service is immediately
+provided to the customer, e.g. digital services.
+
+* Idempotency-Key - An idempotency header ensures that, if the merchant retries a request with the same idempotency key, the retried request will not make additional changes.
+
+* Merchant - A sale unit that integrates with Vipps for online commerce.
+
+* Merchant Serial Number (MSN) - A unique id given to a sale unit by Vipps. This is used to identify a merchant in API logs.
```

Comment:
It is used for a bit more, e.g the point below

Reply:
@Haldormyre Please have a look now :-)

Reply:
That seems great!